students/jack_anderson/mailroom/mailroom.py

```diff
@@ -16,56 +16,88 @@
     ['Lacey Coffin Greene',[7500.32, 6625, 1305, 3400.78]]
     ]
 
+
```

Comment:
You'll want whitespaces between the keys and values:

['Lacey Coffin Greene', [7500.32, etc, etc.